tests/health_checks/NDv4/test-bandwidth.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+#Lock the graphics clock.
+sudo nvidia-smi -lgc 1400 > /dev/null
+
+#Count the GPUs.
+ngpus=$(nvidia-smi --query-gpu=name --format=csv,noheader | wc -l)
+
+#Download the host to device and device to host test from superbench v0.5.
+string="https://raw.githubusercontent.com/microsoft/superbenchmark/release/0.5"\
+"/superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu"
+wget -q $string
+
+#Compile the gpu-copy benchmark.
+nvcc -I/usr/include/ -L/usr/lib/x86_64-linux-gnu/ -lnuma gpu_copy.cu -o gpu-copy
+
+#Run the superbench device to host bandwidth test.
+x_htod=$(./gpu-copy --size 134217728 --num_warm_up 5 \
```

Comment:
Let's check the return status of the test first. Same as L22.

Reply:
Done!